spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -724,89 +700,80 @@ class CometSparkSessionExtensions
       }
 
       // We shouldn't transform Spark query plan if Comet is disabled.
-      if (!isCometEnabled(conf)) return plan
-
-      if (!isCometExecEnabled(conf)) {
-        // Comet exec is disabled, but for Spark shuffle, we still can use Comet columnar shuffle
-        if (isCometShuffleEnabled(conf)) {
-          applyCometShuffle(plan)
-        } else {
-          plan
-        }
```

Comment:
I think with native execution gets more mature, it will be very rare we would like to disable native execution but trigger Comet shuffle. This is not necessary anymore so delete it to make code clear.